packages/clerk-js/src/core/clerk.ts

```diff
@@ -331,6 +325,19 @@ export default class Clerk implements ClerkInterface {
       ...options,
     };
 
+    if (this.#options.sdkMetadata) {
+      Clerk.sdkMetadata = this.#options.sdkMetadata;
+    }
+
+    this.#telemetry = new TelemetryCollector({
+      clerkVersion: Clerk.version,
+      verbose: true,
+      samplingRate: 1,
+      // @ts-expect-error -- in v5 this will always be defined
+      publishableKey: this.publishableKey,
+      ...this.#options.telemetry,
+    });
```

Comment:
Moved the `TelemetryCollector` instantiation into `Clerk.load()` instead of the constructor. It's much easier to pass options this way.